js_modules/dagit/packages/core/src/app/AppCache.tsx

```diff
@@ -17,6 +22,8 @@ export const createAppCache = () =>
         return 'Instance';
       } else if (object.__typename === 'Workspace') {
         return 'Workspace';
+      } else if (object.__typename === 'PartitionBackfill') {
```

Comment:
Oh wow didn't even realize we could do this. It would be nice if we had a helper or something so that we don't need to stuff all of the types in this file which is very general. (eg. `registerDataIDForObject(typename, fn)`)

Reply:
Yeah I definitely agree ... I think we should really have the GraphQL API serve "id" as the "id" field instead of runId, backfillId, etc. I know for runs we went back and added "id" so that we didn't have to do this, and that may be what we should do in the whole system :-/